Documentation/ApiOverview/PasswordPolicies/_PrivateKeyGeneratorCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyAction;
+use TYPO3\CMS\Core\PasswordPolicy\PasswordPolicyValidator;
+
+#[AsCommand(
+    name: 'myextension:generateprivatekey',
+    description: 'Generates an encrypted private key',
+)]
+final class PrivateKeyGeneratorCommand extends Command
+{
+    // inject $myService
```

Comment:
Why not add inject it for a complete example?

Reply:
Because that is a Service you would have to implement, we dont have it, this is beyond the scope of this example?

Reply:
Yup of course, but you can add something like:

```php
public function __construct(private readonly MyService $myService)
{
    parent::__construct();
}
```

That's it.

Reply:
done